sdk/lib/core/uri.dart

```diff
@@ -459,21 +499,23 @@ abstract interface class Uri {
   /// The returned scheme is canonicalized to lowercase letters.
   String get scheme;
 
-  /// The authority component.
+  /// The authority (hostname) component.
   ///
-  /// The authority is formatted from the [userInfo], [host] and [port]
-  /// parts.
+  /// The authority is the [hostname] + [port]. Optionally includes
+  /// user credential ([userInfo]) used to login to the machine.
+  /// It is formatted as [userInfo]@[hostname]:[port]
+  /// Example: leetHaxor@google.com:1337
   ///
-  /// The value is the empty string if there is no authority component.
+  /// If there is no authority component, it is an empty string.
```

Comment:
```dart
   /// If this URI has no authority component ([hasAuthority]), 
   /// this value is the empty string.
   /// Otherwise it is a string starting with `"//"`.
```